src/create-key.ts

```diff
@@ -14,12 +13,24 @@ const deepSortObject = (opts?: any): any => {
     }, {});
 };
 
+function padStart(hash: string, len: number) {
+  while (hash.length < len) {
+    hash = '0' + hash;
+  }
+  return hash;
+}
+
```

Comment:
Could you cite a reference in a comment for where you discovered this hash algo?